client/state/data-layer/wpcom/sites/jitm/index.js

```diff
@@ -31,10 +32,20 @@ const process = {
 	lastSite: null,
 };
 
+/**
+ * Existing libraries do not escape decimal encoded entities that php encodes, this handles that.
+ * @param {string} str The string to decode
```

Comment:
nitpick, but it's nice to have an empty line here between the description and the var types